docs/asciidoc/back_matter.adoc

```diff
@@ -39,3 +37,94 @@ This spec defines a standard **Data Repository Service (DRS) API** (“the yello
+
+(i) The client makes a GET request to identifiers.org to find information about the prefix:
+
+  GET https://registry.api.identifiers.org/restApi/namespaces/search/findByPrefix?prefix=drs.42
+
+This request returns a JSON structure including various URLs containing an embedded namespace id, such as:
+
+  "namespace" : {
+   "href":"https://registry.api.identifiers.org/restApi/namespaces/1234"
+  }
+
+(ii) The client extracts the namespace id (in this example 1234), and uses it to make a second GET request to identifiers.org to find information about the namespace:
+
+  GET https://registry.api.identifiers.org/restApi/resources/search/findAllByNamespaceId?id=1234
+
+This request returns a JSON structure including an urlPattern field, whose value is an URL pattern containing a `${id}` parameter, such as:
+
+  "urlPattern" : "https://drs.myexample.org/ga4gh/drs/v1/objects/{$id}"
+
+=== Calling the n2t.net API as a Client
+
+It takes one API call to get the URL pattern.
+
+The client makes a GET request to n2t.net to find information about the namespace. (Note the trailing colon.)
+
+  GET https://n2t.net/drs.42:
+
+This request returns a text structure including a redirect field, whose value is an URL pattern containing a `$id` parameter, such as:
+
+   redirect: https://drs.myexample.org/ga4gh/drs/v1/objects/$id
+
+=== Caching with Compact Identifiers
+
+Identifiers.org/n2t.net compact identifier resolver records do not change frequently.  This reality is  useful for caching resolver records and their URL patterns for performance reasons.  Builders of systems that use compact identifier-based DRS URIs should cache prefix resolver records from identifiers.org/n2t.net and occasionally refresh the records (such as every 24 hours).  This approach will reduce the burden on these community services since we anticipate many DRS URIs will be regularly resolved in workflow systems.  The implementation of the caching approach is up to system builders but we recommend at least caching the mapping of prefixes to URL patterns. System builders may also choose to cache the compact identifier accession to final DRS URL records as well, but that decision/design is at the discretion of the system builder.  Alternatively, system builders may decide to directly mirror the registries themselves, instructions are provided on the identifiers.org/n2t.net websites.
+
+=== Security with Compact Identifiers
+
+As mentioned earlier, identifiers.org/n2t.net performs some basic verification of new prefixes and provider code mirror registrations on their sites.  However, system builders may have certain security compliance requirements and regulations that prohibit relying on an external site for resolving compact identifiers.  In this case, systems under these security and compliance constraints may wish to whitelist certain compact identifier resolvers and/or vet records from identifiers.org/n2t.net before enabling in their system.
```

Comment:
I like the caveat here about giving up control over security, but I don't understand what guidance we're providing to whom. For server builders, they should either decide they're comfortable with CI-based URIs or not -- that's a one-time decision, and doesn't give them any control over whitelisting or mirroring. For client builders, if they're not allowed to access external sites, I think that means they can't handle CI-based URIs (which technically means they're no longer DRS-compliant) -- what are we suggesting they whitelist, and how would that help?

Reply:
I rephrased this to be "builders of systems that consume and resolve DRS URIs".  Specifically, I added this language in following a conversation with David Bernick at the Broad about the ability of FedRAMP/FISMA Moderate environments that will wish to support DRS 1.1 URI resolution within their security boundary.  By adding this language I'm hoping to give systems like these some guidance on maintaining their compliance while still supporting DRS 1.1